File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1190.py
e = input()
soma=0
for i in range(0,12):
    for j in range(0,12):
        v=float(input())
        if(j>i and i+j>11):
            soma+=v
if(e=='S'):
    print('%.1f' % (soma))
else:
    print('%.1f' % (soma/30))

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1190.cpp
#include <iostream>

using namespace std;

int main(){
    char escolha;
    double v, soma=0;
    cin >> escolha;
    for(int i=0;i<12;i++){
        for(int j=0;j<12;j++){
            cin >> v;
            if(i+j>11 and j>i){
                soma+=v;
            }
        }
    }
    cout.precision(1);
    cout.setf(ios::fixed);
    if(escolha=='S'){
        cout << soma << endl;
    } else if(escolha=='M'){
        cout << soma/30 << endl;
    }

    return 0;
}